tests/functionnal/Entity.php

```diff
@@ -42,6 +42,11 @@ class Entity extends DbTestCase {
       'testChangeEntityParentCached'
    ];
 
+   public function beforeTestMethod($method) {
+      parent::beforeTestMethod($method);
+      $this->login();
+   }
```

Comment:
Are all methods requiring a login ?

Reply:
Probably not on this one, think problem was for ones using cache here. I'll check

Reply:
Changed also